### PR TITLE
Load sample flag to return sample units

### PR DESCRIPTION
**Pull request by LukeLoze1**
During Performance Tests of 30 million (would have been the same for real run) the sample_load fails as it stores all loaded samples in memory and returns them.

This has added a flag to load_sample_file store_loaded_sample_units that is defaulted to False.  If this is True then they will be stored in memory and then returned (for ATs) if not then they won't be stored in memory and large scale Perf Tests will be unlocked

To Test:
    There is a new unit test that 'proves' it.  There are updates to Perf and ATs on this ticket that will bring in the new release v1.0.1.  If you wish to test them with this branch change their pipefile to census-rm-sample-loader = {editable = true,git = "https://github.com/ONSdigital/census-rm-sample-loader", ref = "load-sample-flag-to-return-sample-units”}, then pipenv install --dev

Link:
  https://trello.com/c/luQsADkR/640-enhance-sampleloader-script-8

### CODE REVIEW

**Review by ryangrundy7 — APPROVED** [review]
LGTM 👍

File: load_sample.py [hunk 1]
```diff
@@ -44,9 +48,10 @@ def _load_sample_units(action_plan_id: str, collection_exercise_id: str, sample_
                                                      action_plan_id=action_plan_id),
                                    content_type='application/json')
 
-            sample_unit = {
-                f'sampleunit:{sample_unit_id}': _create_sample_unit_json(sample_unit_id, sample_row)}
-            sample_units.update(sample_unit)
+            if store_loaded_sample_units:
+                sample_unit = {
+                    f'sampleunit:{sample_unit_id}': _create_sample_unit_json(sample_unit_id, sample_row)}
```
Comment (NickGrantX) [inline load_sample.py:53]: you know this isn't JSON, right? I don't really understand why (or how) it works like this.

**Review by NickGrantX — CHANGES_REQUESTED** [review]
Minor comment, but it works so I'll probably just approve it. Just seems really odd to have a colon in the middle of a JSON element name

**Review by NickGrantX — APPROVED** [review]
Sod it. I'm approving. I might not understand the legacy stuff, but it's not your job to fix/tinker.

**Pull request merged**